### PR TITLE
Fixed typo "Foward Slash"

### PR DESCRIPTION
**Pull request by royvanv**

### CODE REVIEW

**Review by ekwoka — APPROVED** [review]

**Comment by royvanv** [comment]
Unfortunately the tests failed. Should we wait for them to be fixed before this PR can be merged?

**Comment by ekwoka** [comment]
@royvanv There are a few flakey tests, especially in the UI tests. 

I think we can both say it's pretty clear this change is not impacting any of those.

but Caleb is quite behind on reviewing PRs, so who knows when it will get merged.

**Comment by joshhanley** [comment]
Thanks for being willing to contribute! This has been fixed in PR #3217 so going to close.

**Pull request closed without merging**
